dotnet/src/Connectors/Connectors.Ollama/Services/OllamaTextGenerationService.cs

```diff
@@ -30,7 +31,7 @@ public OllamaTextGenerationService(
         string modelId,
         Uri endpoint,
         ILoggerFactory? loggerFactory = null)
-        : base(modelId, endpoint, null, loggerFactory)
+        : base(modelId, endpoint, null, loggerFactory?.CreateLogger(typeof(OllamaTextGenerationService)))
```

Comment:
I wonder if you could use `CreateLogger<T>` here?

Reply:
It can, thanks for the suggestion, I`m closing this PR in favor of

- #9488